app/presenters/widget_presenter.rb

```diff
@@ -21,36 +22,36 @@ def render_partial
                                  :layout => false, :locals => {:presenter => self}).html_safe
   end
 
-  def button_fullscreen
-    if @widget.content_type == "chart"
-      @view.link_to(@view.content_tag(:span, '', :class => 'fa fa-arrows-alt fa-fw') + _(" Full Screen"),
-                    {:action => "report_only",
-                     :type   => "hybrid",
-                     :rr_id  => @widget.contents_for_user(current_user).miq_report_result_id},
-                    :id                => "w_#{@widget.id}_fullscreen",
-                    :title             => _("Open the chart and full report in a new window"),
-                    "data-miq_confirm" => _("This will show the chart and the entire report " \
-                                            "(all rows) in your browser. Do you want to proceed?"),
-                    :onclick           => "return miqClickAndPop(this);")
+  def get_html_options(widget_content_type)
```

Comment:
I don't see why Rubocop does not complain about the `get_` prefix for the method.

Generally in Ruby we prefer not to prefix methods with `get_` or `set_`.